apps/routes/lib/route.ex

```diff
@@ -235,6 +235,11 @@ defmodule Routes.Route do
   def combined_route?(%{id: "627"}), do: true
   def combined_route?(_), do: false
 
+  @spec is_rail_route?(t()) :: boolean
+  def is_rail_route?(route) do
```

Comment:
nitpick: In Elixir, `is_X` are for guards, and don't have a `?` (for example, `is_nil/1`).  non-guard boolean functions don't have `is` but do have the `?` (for example, `Enum.empty?/1` or the above `combined_route?/1`)
```suggestion
  @spec rail?(t()) :: boolean
  def rail?(route) do
```